drop-transfer/src/service.rs

```diff
@@ -137,15 +138,21 @@ impl Service {
 
             self.state
                 .event_tx
-                .send(Event::OutgoingTransferFailed(xfer.clone(), err, true))
+                .send((
+                    Event::OutgoingTransferFailed(xfer.clone(), err, true),
+                    utils::current_timestamp(),
+                ))
                 .expect("Event channel should be open");
```

Comment:
TBH it is asking itself for a refactor into a separate method on `State::emit(&self, event: crate::Event)` and then the `utils::current_timestamp()` can be dropped altogether since it would be used in only one place if I'm not wrong

Reply:
the `State::emit` does sound pretty nice, but the `utils` part would have to stay, since the Transfer Request/Queue events still need to use it